internal/project/golang/build.go

```diff
@@ -105,10 +105,15 @@ func (build *Build) CompileDockerfile(output *dockerfile.Output) error {
 		output.Stage(name).
 			From("scratch").
 			Step(step.Copy("/"+name, "/"+name).From(fmt.Sprintf("%s-build", name)))
+
+		all.Step(step.Copy("/", "/").From(name))
 	}
 
+	all := output.Stage(fmt.Sprintf("%s-all", build.Name())).
+		From("scratch")
+
 	for _, artifact := range build.getArtifacts() {
-		addBuildSteps(artifact.name, artifact.config)
+		addBuildSteps(artifact.name, artifact.config, all)
```

Comment:
This looks strange - maybe we can just add another iteration with `build.getArtifacts()` results?

Reply:
I'm not sure what exactly looks strange here? I can do another loop, it doesn't make any difference to me

Reply:
Oh - sorry. What I meant is that you adjusted `addBuildSteps` function, but the new argument is only used in once place at the end of the function. I like new variant with additional `for`, but I think that we can reuse `getArtifacts` result, but its just a nit.